client/blocks/eligibility-warnings/index.jsx

```diff
@@ -57,6 +52,11 @@ function getHoldMessages( translate ) {
 			description: translate( 'Your site must use custom domain in order for this feature to work.' ),
 			supportUrl: 'https://support.wordpress.com/'
 		},
+		NOT_DOMAIN_OWNER: {
+			title: translate( 'Not a custom domain owner' ),
+			description: translate( 'You must be an owner of primary domain subscription in order for this feature to work.' ),
```

Comment:
two little wording tweaks (change "an" to "the" and add another "the")
> You must be the owner of the primary domain subscription [...]

And with that I think this is good to merge.